### PR TITLE
[Doc]add CloudCanal doc

### PR DESCRIPTION
**Pull request by mengtingluo**
Signed-off-by: pudding <a598528009@qq.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [x] This pr needs user documentation (for new or modified features or behaviors)
  - [x] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by hellolilyliuyi — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/loading/CloudCanal.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+CloudCanal help users build a modern data stack in a simple way.
+![image.png](../assets/3.11-1.png)
+
+## Download
+
+[CloudCanal Download Link](https://www.cloudcanalx.com)
+
+[CloudCanal Quick Start](https://www.cloudcanalx.com/us/cc-doc/quick/quick_start)
+
+
```
Comment (evelynzhaojie) [inline docs/loading/CloudCanal.md:15]: extra blank line

File: docs/loading/CloudCanal.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  | Greenplum                  | Yes | Yes | No | Yes |
+  | MySQL                      | Yes | Yes | Yes | Yes |
+  | Kafka                      | No | No | Yes | No |
+  | OceanBase                  | Yes | Yes | Yes | Yes |
+  | PolarDb for MySQL          | Yes | Yes | Yes | Yes |
+  | Db2                        | Yes | Yes | Yes | Yes |
+
+## Typical example
+
+CloudCanal offers a comprehensive productized capability where users can seamlessly add DataSources and create DataJobs through a visual interface. This enables automated schema migration, full data migration, and real-time incremental synchronization. The following example demonstrates how to migrate and synchronize data from MySQL to StarRocks. The procedures are similar for data synchronization between other data sources and StarRocks.
```
Comment (evelynzhaojie) [inline docs/loading/CloudCanal.md:38]: change the first sentence to CloudCanal allows users to perform operations in a visual interface.

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Pull request merged**